tortoise/models.py

```diff
@@ -250,14 +252,18 @@ def db(self):
         )
 
     def get_filter(self, key):
-        filter_info = self.filters[key]
-        overridden_operator = self.db.executor_class.get_overridden_filter_func(
-            filter_func=filter_info['operator'],
-        )
-        if overridden_operator:
-            filter_info = deepcopy(filter_info)
-            filter_info['operator'] = overridden_operator
-        return filter_info
+        return self.filters[key]
+
+    def override_filters(self):
```

Comment:
May be it should be named something like `generate_filters`? Cause in the end it's not just overrides them, but it basically generates values for `filters`  field. And `_filters` could be renamed to `_generic_filters` or `_default_filters` to point out that it is not the same field with `filters`.
It's just that current names made me confused for a while to figure out why there is two of them and how are they connected

Reply:
That is a good point.